src/components/SearchPage.vue

```diff
@@ -270,6 +274,19 @@ export default {
         },
       });
     },
+    createQueryCollection() {
+      const collectionName =
+        Math.random().toString(36).substring(2, 15) + Math.random().toString(36).substring(2, 15);
```

Comment:
use: 
const uuid = require('uuid');

and then:
const collectionName = uuid.v4();